deno/lib/__tests__/pickomit.test.ts

```diff
@@ -85,6 +85,7 @@ test("pick a nonexistent key", () => {
 
   const pickedSchema = schema.pick({
     a: true,
+    // @ts-expect-error
     doesntExist: true,
```

Comment:
this now throws a ts error, which is proof that the change did work. But I believe we should still do some typescript testing like `tsd`..